### PR TITLE
Feature/search bar

### PR DESCRIPTION
**Pull request by blendaf**

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -38,7 +37,10 @@ const GlobalStyle = createGlobalStyle`
 	}
 	
 	* {
+		font-weight: 300;
 		box-sizing:border-box;
+
+
```
Comment (juliuscc) [inline pages/_app.js]: Is there a reason for adding whitespace here?

File: pages/_app.js [hunk 2]
```diff
@@ -22,7 +22,6 @@ const GlobalStyle = createGlobalStyle`
 		margin: 0;
 		background-color: ${props => props.theme.colors.white};
 		color: ${props => props.theme.colors.text};
-		/* font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif; */
```
Comment (juliuscc) [inline pages/_app.js:25]: Nice fix! This has been an eyesore to me.

File: presentational/fragments/Container.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import styled from 'styled-components'
 import screenSizes from '../../utils/screen-sizes'
 
 const Container = styled.div`
-  width: 80%;
+  width: 40%;
```
Comment (juliuscc) [inline presentational/fragments/Container.js]: Does this impact other parts of the page? Are we not going to use a wide container? Could it make more sense to provide two versions of the container: one wide and one slim? Or have a property like 
`<Container wide={true|false} />`

Comment (blendaf) [reply presentational/fragments/Container.js]: Yes this is valid, adjusted the width depending on the prop wide

File: pages/_app.js (hunk 1 above)
Comment (blendaf) [reply pages/_app.js]: Fixed

**Review by juliuscc — APPROVED** [review]

**Pull request merged**